CHANGELOG.md

```diff
@@ -3,6 +3,112 @@
 All notable changes to this project are documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## RMagick 3.2.0
+
+This is expected to be the final deprecation release before RMagick 4.0. We
+have added loads of deprecation warnings to clear the pathway for ImageMagick 7
+support. Once you've fixed all of them you should be fine to upgrade to version
+4.0 without any pain. There are a small handful of edge cases which we could
+not cleanly deprecate, though they should be extremely rare. These will be
+documented in the 4.0 release. Thanks to @dlemstra for the hard work making
+this release possible.
+
+The biggest change in moving towards RMagick 4.0 will be the fact that
+`opacity` is deprecated in favor of `alpha`.
+
+**NOTE: `opacity` is the opposite of `alpha`!!!**
+
+If you are currently passing opacity into methods, you will need to invert the
+value and use the new `alpha:` keyword argument. If you are passing an integer,
+`alpha = 255 - opacity`. An integer `opacity` of 0 is an `alpha` of 255. If you
```

Comment:
The percentage values have not been inverted so this remark can be removed.

Reply:
Wait, so do the percentages still represent opacity?

Reply:
They represent the transparency percentage instead.

Reply:
Ah, I see. So they've always been transparency? Updated.

Reply:
Yup!